File: DevExpress-Demo/CS/DialogsDemo.Wpf/ViewModels/ThemedWindowModule/HeaderModels/WindowTitleEditorHeaderItemModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DialogsDemo {
    public class WindowTitleEditorHeaderItemModel : BaseHeaderItemModel {
        public virtual string Title { get; set; }

        public WindowTitleEditorHeaderItemModel() { ResourceKey = "WindowTitleEditor"; }

        protected void OnTitleChanged() { BaseModel.Title = Title; }
    }
}

File: DevExpress-Demo/VB/DialogsDemo.Wpf/ViewModels/ThemedWindowModule/HeaderModels/WindowTitleEditorHeaderItemModel.vb
Namespace DialogsDemo

    Public Class WindowTitleEditorHeaderItemModel
        Inherits BaseHeaderItemModel

        Public Overridable Property Title As String

        Public Sub New()
            ResourceKey = "WindowTitleEditor"
        End Sub

        Protected Sub OnTitleChanged()
            BaseModel.Title = Title
        End Sub
    End Class
End Namespace
